packages/stores/src/providers/P2PSettingsProvider.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode } from 'react';
+import { useSubscription } from '@deriv/api';
+import P2PSettingsContext, { TP2PSettings } from '../stores/P2PSettingsContext';
+import { useLocalStorage } from 'usehooks-ts';
+
+type TP2PSettingsProvider = {
+    children: ReactNode;
+};
+
+const P2PSettingsProvider = ({ children }: TP2PSettingsProvider) => {
+    const { data, subscribe, ...rest } = useSubscription('p2p_settings');
+    const [p2p_settings, setP2PSettings] = useLocalStorage<TP2PSettings>('p2p_settings', {} as TP2PSettings);
```

Comment:
Do we need to store it in localStorage? Still thinking this can be handled with useMemo instead of useEffect tho

Reply:
maybe we can use useState instead, true we don't need to store it, but I'll try with useState

Reply:
ok both couldn't work 😓 will leave it as localStorage